src/data/messages/messages.json

```diff
@@ -8950,14 +8993,21 @@
                     "options": {},
                     "type": "bool",
                     "name": "unstable",
-                    "id": 50001
+                    "id": 52001
                 },
                 {
                     "rule": "optional",
                     "options": {},
                     "type": "uint32",
                     "name": "wire_type",
-                    "id": 50002
+                    "id": 52002
+                },
+                {
+                    "rule": "optional",
+                    "options": {},
+                    "type": "bool",
+                    "name": "include_in_bitcoin_only",
+                    "id": 60000
```

Comment:
Do you know what was the reason that ids were changed in fw? And also that until the last update it was a sequence from 5000 to 5008 and now it is jumping by hundreds?